src/main/scala/edu/gemini/gsp/sbtplugin/GspPlugin.scala

```diff
@@ -63,9 +62,6 @@ object LucumaPlugin extends AutoPlugin {
 
   import autoImport._
 
-  override def requires: Plugins =
```

Comment:
Why was this removed?

Reply:
It's apparently only required when you need to specify a particular initialization order, otherwise it's unnecessary. I removed it and there were no obvious issues, but I can put it back in if you think we need it.

Reply:
Nah, I was just curious

Reply:
Do you have an opinion on this? When I first wrote the plugin I just copied an example without understanding what was going on. This PR continues that trend by copying from another example that doesn't do it.

Reply:
Not really. I just thought it was required but if not, then better

Reply:
Hey! _I_ wrote the plugin by copying an example without understanding what was going on!